openreview/venue/venue.py

```diff
@@ -384,17 +384,36 @@ def create_registration_stages(self):
     def setup_post_submission_stage(self, force=False, hide_fields=[]):
         venue_id = self.venue_id
         submissions = self.get_submissions()
+        final_hide_fields = ['authors', 'authorids'] + hide_fields
```

Comment:
Are we always hiding authors and authorids?

Reply:
good catch I forgot to check if the venue is double blind.

Reply:
fixed.